doc/toolbox-help.1.md

```diff
@@ -0,0 +1,31 @@
+% toolbox-help(1)
```

Comment:
Oh, one more thing. We also need to update the Bash completion in `completion/bash`.